### PR TITLE
Implement .env support

### PR DESCRIPTION
**Pull request by MikolaLvivUA**
+ add dotenv lib
+ create envFiles
+ add envValues to config files

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DATABASE_NAME = QuestionDB
+DATABASE_USER = mongo user
+DATABASE_PASS = mongo pass
```
Comment (VictorKmin) [inline .env.example]: DATABASE_PASS = 
Example file must be empty

File: src/database/models/question.schema.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Document, Model, model, Schema } from 'mongoose';
-
 import { config } from '../../configs';
+
```
Comment (VictorKmin) [inline src/database/models/question.schema.ts]: rm

File: .env.example (hunk 1 above)
Comment (MikolaLvivUA) [reply .env.example]: fixed

File: src/database/models/question.schema.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/database/models/question.schema.ts]: fixed

**Pull request merged**